### PR TITLE
Add PackageSynchronize command

### PR DESCRIPTION
**Pull request by karniv00l**

### CODE REVIEW

File: src/Command/SynchronizePackageCommand.php [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+    /**
+     * @return void
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('repman:package:synchronize')
+            ->setDescription('Synchronize given package')
+            ->addArgument('package id', InputArgument::REQUIRED, 'package UUID')
```
Comment (akondas) [inline src/Command/SynchronizePackageCommand.php:38]: `packageId` let's stick to one convention

File: tests/Functional/Command/SynchronizePackageCommandTest.php [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+        $commandTester = new CommandTester($this->container()->get(SynchronizePackageCommand::class));
+        $result = $commandTester->execute([
+            'package id' => $packageId,
+        ]);
+
+        /** @var InMemoryTransport $transport */
+        $transport = $this->container()->get('messenger.transport.async');
+
+        self::assertCount(2, $transport->getSent());
```
Comment (akondas) [inline tests/Functional/Command/SynchronizePackageCommandTest.php:39]: why two?

File: tests/Unit/Service/Security/SensioLabsSecurityCheckerTest.php [hunk 3]
```diff
@@ -124,7 +124,7 @@ public function testUpdateWhenRepoDontExist(): void
     public function testThrowErrorWhenUpdateFails(): void
     {
         $this->expectException(ProcessFailedException::class);
-        $this->expectExceptionMessage('repository \'/tmp/repman/security-advisories-repo\' does not exist');
+        $this->expectExceptionMessage("repository '{$this->repoDir}' does not exist");
```
Comment (akondas) [inline tests/Unit/Service/Security/SensioLabsSecurityCheckerTest.php:127]: :+1:

File: tests/Functional/Command/SynchronizePackageCommandTest.php (hunk 2 above)
Comment (karniv00l) [reply tests/Functional/Command/SynchronizePackageCommandTest.php]: there is `$this->fixtures->addPackage()` before, but I've added `$transport->reset()` now so we are checking only what command is doing

**Pull request merged**